### PR TITLE
Operation-app env variables updated

### PR DESCRIPTION
**Pull request by danishjamal104**

### CODE REVIEW

**Comment by danishjamal104** [comment]
@avikganguly01 and @SubhamPramanik please review and merge. Thanks :)

File: helm/ph-ee-engine-oaf/templates/operations-app.yml [hunk 1]
```diff
@@ -35,9 +35,15 @@ spec:
               memory: "512M"
           env:
             - name: "FINERACT_DATASOURCE_CORE_USERNAME"
-              value: "mifos"
+              value: "{{ .Values.operations_app.datasource.username }}" 
             - name: "FINERACT_DATASOURCE_CORE_PASSWORD"
-              value: "password"
+              value: "{{ .Values.operations_app.datasource.password }}"
```
Comment (bisonlou) [inline helm/ph-ee-engine-oaf/templates/operations-app.yml:40]: This looks better as a secret

**Review by bisonlou — CHANGES_REQUESTED** [review]
How about operations web? Does it not need the same variables?

File: helm/ph-ee-engine-oaf/templates/operations-app.yml (hunk 1 above)
Comment (danishjamal104) [reply helm/ph-ee-engine-oaf/templates/operations-app.yml]: updated

**Comment by danishjamal104** [comment]
> How about operations web? Does it not need the same variables?

no there it is not required.

File: helm/ph-ee-engine-oaf/templates/operations-app.yml [hunk 2]
```diff
@@ -35,9 +35,18 @@ spec:
               memory: "512M"
           env:
             - name: "FINERACT_DATASOURCE_CORE_USERNAME"
-              value: "mifos"
+              value: "{{ .Values.operations_app.datasource.username }}"
```
Comment (avikganguly01) [inline helm/ph-ee-engine-oaf/templates/operations-app.yml:38]: Isn't the same changes there in message gateway?

**Review by avikganguly01 — CHANGES_REQUESTED** [review]

File: helm/ph-ee-engine-oaf/templates/operations-app.yml (hunk 2 above)
Comment (danishjamal104) [reply helm/ph-ee-engine-oaf/templates/operations-app.yml:38]: I have added for message gateway also, now check

Comment (danishjamal104) [reply helm/ph-ee-engine-oaf/templates/operations-app.yml:38]: https://github.com/fynarfin/ph-ee-env-labs/pull/26

**Pull request merged**